### PR TITLE
[SAR-10101] Create OTI page

### PR DESCRIPTION
**Pull request by refaque-ahmad**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -103,6 +103,15 @@ tax-identifiers.field_1                                      = Enter a tax ident
 tax-identifiers.field_2                                      = Enter the name of the country that issued the tax identifier
 tax-identifiers.no_identifier_link                           = I do not want to provide an identifier
 
+## Overseas Tax Identifier Page
+tax-identifier.title                                         = Does the business have an overseas tax identifier?
+tax-identifier.line_1                                        = We may use this number to help us identify your business. The tax identifier could be VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify your business details.
```
Comment (HarrNM) [inline conf/messages:108]: The line_1 is missing an 'a' in "The tax identifier could be VAT" compared to the original hint text

File: app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CaptureOverseasTaxIdentifierController.scala [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
+import uk.gov.hmrc.soletraderidentificationfrontend.forms.CaptureOverseasTaxIdentifierForm
+import uk.gov.hmrc.soletraderidentificationfrontend.services.{JourneyService, SoleTraderIdentificationService}
+import uk.gov.hmrc.soletraderidentificationfrontend.utils.MessagesHelper
+import uk.gov.hmrc.soletraderidentificationfrontend.views.html.capture_overseas_tax_identifier_page
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class CaptureOverseasTaxIdentifierController @Inject()(mcc: MessagesControllerComponents,
```
Comment (HarrNM) [inline app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CaptureOverseasTaxIdentifierController.scala:34]: Missing the Singleton above the class

File: conf/messages (hunk 1 above)
Comment (refaque-ahmad) [reply conf/messages:108]: done

File: app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CaptureOverseasTaxIdentifierController.scala (hunk 2 above)
Comment (refaque-ahmad) [reply app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CaptureOverseasTaxIdentifierController.scala:34]: good spot, done

**Review by HarrNM — APPROVED** [review]

**Pull request merged**